cmd/evm/internal/t8ntool/transaction.go

```diff
@@ -183,9 +183,6 @@ func Transaction(ctx *cli.Context) error {
 		if chainConfig.IsShanghai(new(big.Int), 0, 0) && tx.To() == nil && len(tx.Data()) > int(chainConfig.MaxInitCodeSize()) {
 			r.Error = errors.New("max initcode size exceeded")
 		}
-		if chainConfig.IsOsaka(new(big.Int), 0) && tx.Gas() > params.MaxTxGas {
-			r.Error = errors.New("gas limit exceeds maximum")
```

Comment:
Do you think it might be better to just skip when chain is arbitrum like i did here https://github.com/OffchainLabs/go-ethereum/pull/522/files ?

I dont have a strong preference either way, but wanted to get your opinion.

Reply:
Hate to reopen this can of worms.
But, I kind of agree that it would be better to have the code in here.
That way, the nitro fork can still be run as a proper L1 node (when != arbitrum)
Maybe we could rename the `params` variable to something like:
`params.MaxTxGasRenamedForNitroMerges` (or something more concise, but equally clear.

That way, if the upstream code starts using `MaxTxGas` elsewhere, we'll still notice it, but we can also keep the functionality when arbitrum is disabled?